templates/pg_upgrade.sh.j2

```diff
@@ -7,169 +7,98 @@
 # - Paul Bonaud
 # Description:
 #   This helper tries to strictly follow the official documentation of PostgreSQL
-#   from the pg_upgrade page: https://www.postgresql.org/docs/9.5/static/pgupgrade.html
+#   from the pg_upgrade page: https://www.postgresql.org/docs/current/static/pgupgrade.html
 # Usage:
 #   WARNING: Please read both the documentation and this script if you want to run it
-#   1. edit this file and remove the "DRY_RUN=1" line
-#   2. execute this script as the postgres admin user
+#   1. run this script on the postgresql server with a user able to use sudo as root and postgres users
+#   2. edit this file and remove the "exit 1" line to really upgrade the cluster
 
-{% if postgres_pgbouncer_uri is defined and postgres_pgbouncer_uri %}
-PGBOUNCER_PG_URI={{ postgres_pgbouncer_uri }}
-{% endif %}
-DATABASE_NAME={{ postgres_dbname }}
-DATABASE_PORT={{ postgres_new_cluster_port }}
+# Output every line executed and stop on the first error
+set -xe
```

Comment:
Would it be problematic to fail on pipes too?
```suggestion
set -xeo pipefail
```

Reply:
Also, this options were passed from the bash command (in the first line of the file)
Could you change the command from `#!/bin/bash -ex` to `#!/usr/bin/env bash` please? (for better cross-platform compatibility)